docs/modules/ROOT/pages/index.adoc

```diff
@@ -610,17 +617,17 @@ The `signature` field of an *<<_identity_assertion,identity assertion>>* MUST co
 
 The `pad1` and `pad2` fields of an *<<_identity_assertion,identity assertion>>* MUST contain only zero-value (`0x00`) bytes. The `cawg.identity.pad.invalid` error code SHALL be used to report assertions that contain other values in these fields.
 
-==== Validating the `expected_claim` value
+==== Validating the `expected_partial_claim` value
 
-If the *<<_identity_assertion,identity assertion>>* contains an `expected_claim` value in `signer_payload`, it MUST be validated as follows:
+If the *<<_identity_assertion,identity assertion>>* contains an `expected_partial_claim` value in `signer_payload`, it MUST be validated as follows:
```

Comment:
contains an `expected_partial_claim` *field* in `signer_payload`, it MUST be validated as follows: